source/develop/release-management/features/virt/vm-failover-using-vm-leases.html.md

```diff
@@ -0,0 +1,181 @@
+---
+title: VM failover using vm leases
+category: feature
+authors: nsoffer
+feature_name: VM failover using vm leases
+feature_modules: engine
+feature_status: Design
+---
+
+# VM failover using vm leases
+
+
+## Overview
+
+This feature adds the ability to start a highly available VM on another
+host when the host becomes unreachable and the host cannot be fenced.
+In the current system the VM state is unknown and the system cannot
+start the vm on one of the available hosts until the host is rebooted
+manually and the administrator mark the host as rebooted manually.
+
+This feature is tracked in bug
+https://bugzilla.redhat.com/804272
```

Comment:
This is a customer bug for RHV. Let's use the oVirt bug https://bugzilla.redhat.com/1317429.

Also, does middleman render this as a clickable link?